src/components/app-context/AppContext.jsx

```diff
@@ -1,4 +0,0 @@
-import { createContext } from 'react';
```

Comment:
@davidjoy pointed out that this should probably come back since it likely has a different format/data in it than the frontend-platform context